roles/conf/distsearch/tasks/distributedSearch/server.yml

```diff
@@ -9,11 +9,14 @@
 
 # server
 
+
 - name: "Configure distsearch.conf [distributedSearch] - server state=present"
+  vars:
+    searchpeers: "{{ groups['shcmember'] }}, {{ groups['licensemaster'] }}, {{ groups['masternode'] }}"
   ini_file: dest={{ splunk_conf_path }}/distsearch.conf
             section=distributedSearch
             option=servers
-            value="{{ splunk_distsearch_conf.distributedSearch.servers | create_distsearch_servers  }}"
+            value="{{ searchpeers | create_distsearch_servers  }}"
             state=present
   when: splunk_distsearch_conf.distributedSearch.servers is defined
```

Comment:
If servers is defined, why have searchpeers?  Or vice versa?  Seems like this check is pointless/invalid if we're not actually using that var anyway.  And if we are setting it, why aren't we using it?  Was this some sort of workaround for a bug?